StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/SA1639FileHeaderMustHaveSummary.cs

```diff
@@ -33,6 +33,7 @@
     /// </code>
     /// </remarks>
     [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    [NoCodeFix("Cannot generate documentation")]
```

Comment:
:exclamation: If we can fix SA1633, I imagine we could fix this one as well, even if it's not done immediately.